components/LoginDiscoveryForm.tsx

```diff
@@ -40,22 +41,33 @@ const ContinueToTenantForm = ({ onBack }: { onBack: () => void; }) => {
 
 const LoginDiscoveryForm = () => {
   const [isDiscovery, setIsDiscovery] = useState(true);
+  const [googleOAuthURL, setGoogleOAuthURL] = useState("");
+  useEffect(() => {
+      // UseEffect since format requires window to be loaded
+    setGoogleOAuthURL(formatOAuthDiscoveryStartURL("google"));
+  },[]);
 
   if (isDiscovery) {
     return (
-      <EmailLoginForm title="Sign in" onSubmit={discoveryStart}>
-        <p>
-          We&apos;ll email you a magic code for a password-free sign in.
-          <br />
-          You&apos;ll be able to choose which organization you want to access.
-          <br />
-          Or you can{" "}
-          <Link href="" onClick={() => setIsDiscovery(false)}>
-            sign in manually instead
-          </Link>
-          .
-        </p>
-      </EmailLoginForm>
+      <>
+        <EmailLoginForm title="Sign in" onSubmit={discoveryStart}>
+          <p>
+            We&apos;ll email you a magic code for a password-free sign in.
+            <br />
+            You&apos;ll be able to choose which organization you want to access.
+            <br />
+            Or you can{" "}
+            <Link href="" onClick={() => setIsDiscovery(false)}>
+              sign in manually instead
+            </Link>
+            .
+          </p>
+        </EmailLoginForm>
+        or
+        <Link href={googleOAuthURL}>
+          Sign in with Google
```

Comment:
Can we add an icon & style this as a button? It would be nice for it to be equal to the `Continue` button

Reply:
No problem